cost/superseded_instance/superseded_instance.pt

```diff
@@ -467,6 +492,9 @@ policy "pol_aws_superseded_instance" do
       field "new_instance_type" do
         label "New Instance Type"
       end
+      field "has_instance_size_changed" do
+        label "Instance has been adjusted"
```

Comment:
Label should be capitalized like the others since it is the header of the incident table.